Apps/W1/ShopifyConnector/app/Products/Codeunits/ShpfyProductExport.Codeunit.al

```diff
@@ -0,0 +1,636 @@
+/// <summary>
+/// Codeunit Shpfy Product Export (ID 30178).
+/// </summary>
+codeunit 30178 "Shpfy Product Export"
+{
+    Access = Internal;
+    TableNo = "Shpfy Shop";
+
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"